lib/ide/parser/src/jsclient.rs

```diff
@@ -31,13 +31,16 @@ impl From<serde_json::error::Error> for Error {
     }
 }
 
+impl From<JsValue> for Error {
+    fn from(_jsvalue: JsValue) -> Self {
+        Error::ScalaException()
```

Comment:
It would be cool to provide information what kind of error it was. If we ever get this error, we would debug it for hours otherwise.